templates/views/thread-header.html

```diff
@@ -64,6 +64,10 @@
     </div>
     <div class="f-toggle-aside item ui button icon"
          title="Toggle information sidebar">
-        <i class="icon info circle grey"></i>
+        {{#if notices.length}}
```

Comment:
add hasNotices to this view's render_attributes.  Just copy the code used in the aside view's render_attributes.